app/controllers/users.js

```diff
@@ -269,3 +271,68 @@ exports.user = (req, res, next, id) => {
       next();
     });
 };
+
+/**
+ * Invite user to play game.
+ */
+const sgMail = require('@sendgrid/mail');
```

Comment:
Please move this to the top of the file and use a descriptive variable naming convention, such as `sendgridMail` instead of `sgMail` @OlatunjiYso @Rhotimee